internal/store/backup_configs.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+
+	"github.com/mongodb/mongocli/internal/config"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+//go:generate mockgen -destination=../mocks/mock_backup_config.go -package=mocks github.com/mongodb/mongocli/internal/store BackupConfigGetter,BackupConfigLister
+
+type BackupConfigGetter interface {
+	GetBackupConfig(string, string) (*opsmngr.BackupConfig, error)
+}
+
+type BackupConfigLister interface {
+	ListBackupConfigs(string, *atlas.ListOptions) (*opsmngr.BackupConfigs, error)
+}
+
+// GetBackupConfig encapsulates the logic to manage different cloud providers
+func (s *Store) GetBackupConfig(projectID, clusterID string) (*opsmngr.BackupConfig, error) {
+	switch s.service {
+	case config.OpsManagerService:
+		result, _, err := s.client.(*opsmngr.Client).BackupConfigs.Get(context.Background(), projectID, clusterID)
+		return result, err
+	default:
+		return nil, fmt.Errorf("unsupported service: %s", s.service)
+	}
+}
+
+// ListBackupConfigs encapsulates the logic to manage different cloud providers
+func (s *Store) ListBackupConfigs(projectID string, options *atlas.ListOptions) (*opsmngr.BackupConfigs, error) {
+	switch s.service {
+	case config.OpsManagerService:
```

Comment:
[nit] this should work for cloud manager as well, https://docs.cloudmanager.mongodb.com/reference/api/backup-configurations/